### PR TITLE
update build configurations and permisions

### PR DESCRIPTION
**Pull request by mozzy11**

### CODE REVIEW

**Comment by mozzy11** [comment]
@caseyi @CalebSLane ,
This PR should be ready for merge .
and we should build plugins based on these fixes.

Basically i updated the Permision Roles and also fixed the maven build configurations

**Comment by mozzy11** [comment]
All analyzers now display fine 
image

File: README.md [hunk 1]
```diff
@@ -2,3 +2,15 @@ openelisglobal-plugins
...
+
+[![Build Status](https://github.com/openelisglobal/openelisglobal-plugins/actions/workflows/ci.yml/badge.svg)](https://github.com/openelisglobal/openelisglobal-plugins/actions/workflows/ci.yml)
+
+For Building The Plugins 
+
+1. Got to the [Parent pom file](./pom.xml)
+1. set the right absolute `SystemPath path` to the `openelisglobal` dependence located under the [lib](./lib/) folder
+1. Run the Maven Build  
+
+    ```mvn clen install```
```
Comment (CalebSLane) [inline README.md:14]: mvn clean install typo

Comment (mozzy11) [reply README.md:14]: Thanks @CalebSLane for the catch

**Pull request merged**